spec/requests/api/v2/providers/accredited_body_filter_providers_spec.rb

```diff
@@ -0,0 +1,202 @@
+require "rails_helper"
+
+describe "AccreditedBody API v2", type: :request do
+  describe "GET /providers" do
+    let(:user) { create(:user, organisations: [organisation]) }
+    let(:organisation) { create(:organisation) }
+    let(:recruitment_cycle) { find_or_create :recruitment_cycle }
+    let(:payload) { { email: user.email } }
+    let(:token) do
+      JWT.encode payload,
+                 Settings.authentication.secret,
+                 Settings.authentication.algorithm
+    end
+    let(:credentials) do
+      ActionController::HttpAuthentication::Token.encode_credentials(token)
+    end
+
+    let(:physical_education) { find_or_create(:secondary_subject, :physical_education) }
+    let(:biology) { find_or_create(:secondary_subject, :biology) }
+
+    let(:unfunded_pe_course) do
+      create(:course,
+             level: :secondary,
+             provider: delivering_provider1,
+             subjects: [physical_education],
+             site_statuses: [build(:site_status, :findable, site: build(:site))],
+             accrediting_provider: accredited_provider)
+    end
+
+    let(:fee_funded_pe_course) do
+      create(:course,
+             level: :secondary,
+             program_type: :school_direct_training_programme,
+             provider: delivering_provider1,
+             subjects: [physical_education],
+             site_statuses: [build(:site_status, :findable, site: build(:site))],
+             accrediting_provider: accredited_provider)
+    end
+
+    let(:non_pe_course) do
+      create(:course,
+             level: :secondary,
+             provider: delivering_provider2,
+             subjects: [biology],
+             site_statuses: [build(:site_status, :findable, site: build(:site))],
+             accrediting_provider: accredited_provider)
+    end
+
+    let(:delivering_provider1) { create(:provider) }
```

Comment:
We don't seem to have an official name for this kind of provider but we started calling them `training_providers` ([small discussion here](https://ukgovernmentdfe.slack.com/archives/CAHAJFR0X/p1586179199434100?thread_ts=1586179143.434000&cid=CAHAJFR0X)). It's used across the api and Publish. I was wondering if we could rename them to keep things consistent.

Reply:
Thanks @AgaDufrat - I will make the change